src/Materials/shaderMaterial.ts

```diff
@@ -1053,6 +1170,21 @@ export class ShaderMaterial extends Material {
             result.setTexture(key, this._textures[key]);
         }
 
+        // TextureArray
+        for (var key in this._textureArrays) {

```

Comment:
nit - `let` instead of `var`

Reply:
I prefer not changing it because there are 20+ other similar loops in that function that are already existing and that are using `var` and updating the `var` to `const` is not really the goal of the PR (but that could probably be something we should do one day on all the Babylon code base). Only updating this loop to `const` would look strange among the others 20+, that's why I used `var` in the first place.